### DISCUSSION TITLE
Request you to delete repository from my account

### DISCUSSION THREAD

**Discussion by devagekarswati** [community/community#116875] [Repositories] [answered]
###Delete repository

Question

#Delete repository

As I am unable to login to "[redacted by moderator]" account can you delete the repository from my other account "swatidevagekar". If yes, I will share the details .

**Comment by moritztim** [comment, marked as answer]
You need to send an email to github support, this is a public discussion.

**Reply by devagekarswati** [reply]
Could you please help me with github support email id?

**Comment by devagekarswati** [comment]
Could you please help me with the support email id

**Reply by moritztim** [reply]
You can find it I believe in you

**Reply by devagekarswati** [reply]
I tried to search but could not find. Can you please send the mail id

**Reply by moritztim** [reply]
literally google github support. I found this https://support.github.com/

**Reply by devagekarswati** [reply]
I need email id. I have already raised ticket here

**Reply by moritztim** [reply]
I don't think they have just a support email address, they would get flooded by uncategorized emails. If you opened a ticket, you should just wait for them to get back to you

**Comment by User132New** [comment]
Only the owner can delelte the repository.
Try forget password or gain access to recovery methods

**Comment by queenofcorgis** [comment]
Hi @devagekarswati ,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
